tests/system_info_test.py

```diff
@@ -29,7 +29,6 @@ def test_get_deps_info() -> None:
     assert "cudf" in deps_info
     assert "modin" in deps_info
     assert "pyarrow" in deps_info
-    assert "numpy" in deps_info
```

Comment:
can we keep numpy? it'd be useful to have it to reproduce certain pandas / pyarrow issues